INSTALL.md

```diff
@@ -42,29 +41,26 @@ This requires a MongoDB server to be running on your desired host machine.
 #### Python setup
 
 The next step is to set up a Python environment that contains all of the required dependencies with the correct versions.
-You will need Python 3.10 or higher to run *datalab*; we recommend using
-something like [`pyenv`](https://github.com/pyenv/pyenv) to manage Python versions on your machine, to avoid breakages based on your OS's Python versioning.
+You will need Python 3.10 or higher to run *datalab*; we recommend using a tool
+to manage Python versions on your machine, to avoid breakages based on your OS's Python versioning (e.g., [`pyenv`](https://github.com/pyenv/pyenv) or [`uv`](https://github.com/astral-sh/uv)).
 
-##### Using virtual environments with `uv` or `venv`
+##### Installation with `uv` or `venv`
 
-We recommend using a virtual environment tool of your choice to manage the dependencies for the
-Python server, for example [`uv`](https://github.com/astral-sh/uv) (see
-repository for installation instructions), or the
-standard library Python `venv` module.
+We recommend using [`uv`](https://github.com/astral-sh/uv) (see the linked repository or https://docs.astral.sh/uv for installation instructions) for managing your *datalab* installation.
+
+You could also use the standard library `venv` module, but this will not allow you to install pinned dependencies as easily, and is significantly slower than `uv`.
 
 1. Create a virtual environment for *datalab*, ideally inside the `pydatalab` directory.
-    - For `uv`, this can be done with `uv venv`.
+    - For `uv`, this will be done automatically on installation.
     - For `venv`, this can be done with `python -m venv .venv`.
-    - You will be left with a folder called `.venv` that bundles a Python
-      environment.
-2. Activate the virtual environment (optional for `uv`) and install dependencies. One can either use the loosely pinned dependencies in `pyproject.toml`, or the locked versions in the `requirements/requirements-all-dev.txt` and `requirements/requirements-all.txt` files.
+    - Either way, you will be left with a folder called `.venv` in your `pydatalab` directory that bundles an entire Python environment.
+2. Activate the virtual environment (again, optional for `uv`) and install dependencies. One can either use the loosely pinned dependencies in `pyproject.toml`, or the locked versions in `uv.lock`.
 
 === "Installation with `uv`"
 
     ```shell
-    # EITHER: Install all dependencies with locked versions, then install the local package
-    uv pip install -r requirements/requirements-all-dev.txt
-    uv pip install -e '.[all,dev]'
+    # EITHER: Install all dependencies with locked versions
+    uv sync --all-extras --dev
     # OR: Install all dependencies with loosely pinned versions
     uv pip install -e '.[all,dev]'
```

Comment:
This line gives a warning because `dev` doesn't exist now, and I guess the dev requirements from don't get installed when using this option.